collectors/python.d.plugin/example/example.conf

```diff
@@ -66,3 +66,26 @@
 # ----------------------------------------------------------------------
 # AUTO-DETECTION JOBS
 # only one of them will run (they have the same name)
+
+four_lines:
+    name: "Four Lines"      # the JOB's name as it will appear at the
+    update_every: 1         # the JOB's data collection frequency
+    priority: 60000         # the JOB's order on the dashboard
+    penalty: yes            # the JOB's penalty
+    autodetection_retry: 0  # the JOB's re-check interval in seconds
+    # some collector specific params
+    num_lines: 4            # the number of lines to create
+    lower: 0                # the lower bound of numbers to randomly sample from
+    upper: 100              # the upper bound of numbers to randomly sample from
```

Comment:
I think you should add new parameter descriptions after L60 and remove `# some collector specific params`.